calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -412,6 +449,47 @@ public CalendarEventVector getEvents(List references, TimeRange range)
 		return this.getEvents(references, range, false);
 	}
 
+	public List<CalendarEvent> getFilteredEvents(Map<EventFilterKey, Object> options) {
+
+        options = options == null ? Collections.emptyMap() : options;
```

Comment:
I think I prefer a 'no-brackets if' than assigning a variable to itself like that :/

Wrong tabbing too.

Reply:
```suggestion
        if (options == null) { options = Collections.emptyMap(); }
```

Best of both worlds: still one line, with braces, no assignment to itself.